cross/boost/Makefile

```diff
@@ -1,15 +1,15 @@
 PKG_NAME = boost
-PKG_VERS = 1.63.0
```

Comment:
Please try again with python included but without upgrade... if not required by i2pd